src/outagesim/backup_reliability.jl

```diff
@@ -91,14 +172,51 @@ julia> starting_probabilities(2, 0.99, 0.05)
  0.00354025  0.11192  0.88454
 ```
 """
-function starting_probabilities(N::Int, OA::Real, failure_to_start::Real)::Matrix{Float64} 
-    M = markov_matrix(N, (1-OA) + failure_to_start*OA) 
-    G = hcat(zeros(1, N), 1)
+function starting_probabilities(num_gen::Int, gen_operational_availability::Real, gen_failure_to_start::Real)::Matrix{Float64} 
+    M = markov_matrix(num_gen, (1-gen_operational_availability) + gen_failure_to_start*gen_operational_availability) 
+    G = hcat(zeros(1, num_gen), 1)
+    return G * M
```

Comment:
Why not just index into matrix i.e. M[end,:] ? Also applies to multiple gen type version of function

Reply:
Applied Change